lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -55,31 +56,42 @@ class LinkBlockingFirstPaintAudit extends Audit {
       };
     }
 
-    let totalSpendTime = 0;
-    const results = artifact.items
-      .filter(item => item.tag.tagName === tagFilter)
-      .map(item => {
-        totalSpendTime += item.spendTime;
+    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
 
-        return {
-          url: item.tag.url,
-          label: `delayed first paint by ${item.spendTime}ms`
-        };
-      });
+    const startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), Number.MAX_VALUE);
```

Comment:
It's possible for that difference to be large (though in practice I suspect it to be small unless people love putting huge script in the head or using document.write all over the place in which case there are other audits for them and I'm happy to punish them a little more than necessary here :) ) given our numbers because these are all purely network times not parse or anything else that would have generated these large gaps, but the goal here is for there to be 0 items blocking first paint and as far as I'm concerned it makes more sense to report that the violations pushed out first paint to at least X ms than to report the share of network time they occupied.